monkey/infection_monkey/exploit/HostExploiter.py

```diff
@@ -58,7 +59,7 @@ def report_login_attempt(self, result, user, password="", lm_hash="", ntlm_hash=
     # TODO: host should be VictimHost, at the moment it can't because of circular dependency
```

Comment:
Did you try running? Is this still valid?

Reply:
Try running BB tests